server/main_server/src/rodsServer.cpp

```diff
@@ -139,21 +139,21 @@ namespace
         // Should only ever set the cache salt once.
         try {
             const auto& existing_salt = irods::get_server_property<const std::string>(irods::KW_CFG_RE_CACHE_SALT);
-            log_server::error("createAndSetRECacheSalt: salt already set [{}]", existing_salt.c_str());
-            return ERROR(SYS_ALREADY_INITIALIZED, "createAndSetRECacheSalt: cache salt already set");
+            log_server::debug("createAndSetRECacheSalt: Cache salt already set [{}]", existing_salt.c_str());
```

Comment:
Is this changing to `debug` because the returned `irods::error` will get logged somewhere?

Reply:
`debug` because this will be the normal case on a reload.

Reply:
Okay